playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -622,6 +622,7 @@ EDXAPP_REGISTRATION_EXTRA_FIELDS:
   mailing_address: "optional"
   goals: "optional"
   honor_code: "required"
+  privacy_policy: "required"
```

Comment:
We override this for CME via secrets.

@caesar2164 Can you test this with CME registration enabled?

I think we may need to make this change to secrets too.